preview/action.yaml

```diff
@@ -10,40 +10,41 @@ inputs:
   project-slug:
     description: "Project's slug on Read the Docs"
     required: true
-  project-language:
+  project-languages:
     description: "Project's language on Read the Docs"
     default: "en"
     required: false
   message-template:
     description: "Template message to use for the PR body"
     default: |
       ----
-      :books: Documentation preview :books:: {docs-pr-index-url}
+      :books: Documentation preview :books::
+      {docs-pr-index-url}
```

Comment:
I'd avoid changing this template to avoid breaking changes as well.

Reply:
If you do not change the default, the following awkward output will appear when multiple languages exist.
```
----
:books: Documentation preview :books:: https://sorna--1139.org.readthedocs.build/ko/1139/
https://sorna--1139.org.readthedocs.build/en/1139/
```

Reply:
I just thought it would be nice to keep the existing formatting and add a line break before each url in case of multiple languages!

Reply:
I just thought it would be nice to keep the existing formatting and add a line break before each url in case of multiple languages!
Also, adding bullets should only be added when there are multiple languages, so it would be nice to branch.